crates/masonry/src/text2/edit.rs

```diff
@@ -147,17 +183,115 @@ impl<T: EditableText> TextEditor<T> {
                             Handled::No
                         }
                     }
+                } else if mods.control_key() || mods.super_key()
```

Comment:
On macOS, I believe that this case specifically actually uses "Option"

Iced calls this [`is_jump_modifier_pressed`](https://github.com/iced-rs/iced/blob/master/widget/src/text_input.rs#L1287)

Of course, we should just use the events from the platform for macOS, but you know. winit

Reply:
Some day... but for today it's TODO